dist/details-group/details-group.css

```diff
@@ -0,0 +1,54 @@
+ul.details-group {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+}
+
+ul.details-group li:not(:last-child) {
+    border-bottom: 1px solid var(--color-stroke-subtle);
+}
+
+ul.details-group summary.details__summary {
+    border-radius: 0;
+    display: flex;
+    font-size: var(--font-size-medium);
+    justify-content: space-between;
+    min-height: 48px;
+    padding: 12px 16px;
+}
+
+ul.details-group details > :not(summary.details__summary) {
+    margin: 0 16px 6px;
+}
+
+ul.details-group details > p {
+    max-width: 600px;
+}
+
+ul.details-group--large summary.details__summary {
+    font-size: var(--font-size-large-1);
+    min-height: 52px;
+}
+
+ul.details-group details span.details__icon {
```

Comment:
Just reiterating/reminding to move this to using two icons with show/hide of each based on `:open`.